Wikipedia/Code/ArticleCollectionViewCell.swift

```diff
@@ -16,11 +16,11 @@ open class ArticleCollectionViewCell: CollectionViewCell, SwipeableCell, BatchEd
     private var _titleBoldedString: String? = nil
     
     private func updateTitleLabel() {
-        if let titleHTML = _titleHTML {
-            let attributedTitle = titleHTML.byAttributingHTML(with: titleTextStyle, matching: traitCollection)
+        if var titleHTML = _titleHTML {
             if let boldString = _titleBoldedString {
-                attributedTitle.applyBoldFont(to: boldString, textStyle: titleTextStyle, matching: traitCollection)
+                titleHTML.applyBoldTag(to: boldString)
             }
+            let attributedTitle = titleHTML.byAttributingHTML(with: titleTextStyle, matching: traitCollection)
```

Comment:
This would previously override the font applied by `byAttributingHTML`. Now, it adds a bold tag so that `byAttributingHTML` will apply the bold (or bold + italic) font